scipy/signal/_filter_design.py

```diff
@@ -2403,14 +2403,14 @@ def iirdesign(wp, ws, gpass, gstop, analog=False, ftype='ellip', output='ba',
             if any(wp >= 1) or any(ws >= 1):
                 raise ValueError("Values for wp, ws must be less than 1")
         elif any(wp >= fs/2) or any(ws >= fs/2):
-            raise ValueError("Values for wp, ws must be less than fs/2"
-                             f" (fs={fs} -> fs/2={fs/2})")
+            raise ValueError("Values for wp, ws must be less than fs/2 "
+                             f"(fs={fs} -> fs/2={fs/2})")
```

Comment:
Nit but here above the quotes should be aligned

Reply:
I'm kinda new to all this, is it okay now? I changed to draft, do I need to submit a new PR?

Reply:
Looking back, I screwed up, let me fix it quickly

Reply:
Really sorry but this was a moment of blindness from me, I think the original was actually more correct. Since this is my fault, I am happy to revert your PR to the original commit and merge if you are happy with things then